airbyte-integrations/connectors/source-github-singer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM airbyte/integration-base-singer:dev
+
+RUN apt-get update && apt-get install -y \
```

Comment:
@jrhizor are these the same for all singer / postgres integrations. could we move this into the base? happy to just do it in this PR if it's the right thing to do.

Reply:
I don't think these should be necessary for most integrations. They should be only included if necessary.